packages/instant/src/components/zero_ex_instant_provider.tsx

```diff
@@ -120,6 +121,15 @@ export class ZeroExInstantProvider extends React.Component<ZeroExInstantProvider
         gasPriceEstimator.getGasInfoAsync();
         // tslint:disable-next-line:no-floating-promises
         this._flashErrorIfWrongNetwork();
+
+        // Analytics
+        analytics.addEventProperties({
```

Comment:
Instead of sprinkling in `.track`s with arbitrary strings throughout the code base, I'm thinking it might make sense to have a centralized helper so we have all event names and properties in one place.  Does this sound reasonable, or like overkill?

Reply:
Don't feel strongly about this

Reply:
After writing out a bunch of events we want to track in the quip doc, I think it'd be worthwhile to centralize all the event names and their properties -- I've seen this be helpful in past projects when dealing with a large amount of events.  I'll work on a commit

Reply:
we should at least have one file with all the eventNames as constants and maybe some way of typing the corresponding eventProperties

Reply:
coo yea i agree, working on this

Reply:
this is done